src/schema/v2/previewSavedSearch.ts

```diff
@@ -102,9 +102,13 @@ const PreviewSavedSearchType = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(
         new GraphQLList(new GraphQLNonNull(SearchCriteriaLabel))
       ),
-      resolve: async (_root, _args, { filterArtworksLoader }) => {
+      resolve: async ({ artistIDs }, _args, { filterArtworksLoader }) => {
+        if (!artistIDs) {
+          throw new Error("artistIDs are required to get suggested filters")
+        }
+
         const suggestedFiltersByArtist: SearchCriteriaLabel[][] = await Promise.all(
-          _root.artistIDs.map((artistSlug) =>
+          artistIDs.map((artistSlug) =>
             getSuggestedFiltersByArtistSlug(artistSlug, filterArtworksLoader)
           )
```

Comment:
I think the same comment applies here as @dblandin [noted already](https://github.com/artsy/metaphysics/pull/5418#discussion_r1395862549), sorry I missed that the first time.

Reply:
nice idea - didn't know that's possible. I will do that